src/atlas4py/_atlas4py.cpp

```diff
@@ -225,14 +238,17 @@ PYBIND11_MODULE( _atlas4py, m ) {
         .def_property( "nodes", py::overload_cast<>( &Mesh::nodes, py::const_ ), py::overload_cast<>( &Mesh::nodes ) )
         .def_property( "edges", py::overload_cast<>( &Mesh::edges, py::const_ ), py::overload_cast<>( &Mesh::edges ) )
         .def_property( "cells", py::overload_cast<>( &Mesh::cells, py::const_ ), py::overload_cast<>( &Mesh::cells ) );
-    m.def( "build_edges", []( Mesh& mesh ) { mesh::actions::build_edges( mesh, option::pole_edges( false ) ); } );
+    m.def( "build_edges", []( Mesh& mesh, std::optional<std::reference_wrapper<eckit::Configuration>> const& config ) {
+        mesh::actions::build_edges( mesh, config ? config.value().get() : noconfig() );
```

Comment:
To me the code would look slightly simpler, if you just do
```suggestion
        if(config)
            mesh::actions::build_edges( mesh, config.value().get());
        else
            mesh::actions::build_edges( mesh);
```
instead of introducing the static object. Especially, since the atlas library has the single argument overload.